### PR TITLE
add -c center mode on wayland

### LINKED ISSUE #196: Adding -c/--center option like dmenu
I like seeing the menu centered on the screen with the list arranged vertically

### PR DESCRIPTION
**Pull request by stacyharper**
depends on : https://github.com/Cloudef/bemenu/pull/197

resolve : https://github.com/Cloudef/bemenu/issues/196

### CODE REVIEW

**Comment by Cloudef** [comment]
Needs to be rebased, would be nice if X11 implementation could be done as well, but not necessary.

**Comment by Eluminae** [comment]
Done !

I'm sorry, I'll only focus on Wayland :(

File: client/common/common.c [hunk 1]
```diff
@@ -171,6 +171,7 @@ usage(FILE *out, const char *name)
           " -F, --filter          filter entries for a given string.\n"
           " -w, --wrap            wraps cursor selection.\n"
           " -l, --list            list items vertically with the given number of lines.\n"
+          " -c, --center          list items in the center of the screen.\n"
```
Comment (Cloudef) [inline client/common/common.c:174]: remove this as this is backend specific option

Comment (Eluminae) [reply client/common/common.c]: You right ! Removed !

**Comment by Cloudef** [comment]
Thanks

**Pull request merged**